nep-17.mediawiki

```diff
@@ -0,0 +1,109 @@
+<pre>
+  NEP: 17
```

Comment:
Since we are on the 17, maybe we should jump 3 and make NEP-20. aheuaheua

Reply:
@vncoelho I thought so too... especially if we are adding approve/transferFrom!

Reply:
It's meaningless.

Reply:
That is true. It would be a good remembering for the general public and is a little bit didactic...aheuahea
Anyway, it is meaningless and would leave here with 3 empty slots.